test/framework/vsphere.go

```diff
@@ -247,13 +242,34 @@ func (v *VSphere) WithNewVSphereWorkerNodeGroup(name string, workerNodeGroup *Wo
 }
 
 func (v *VSphere) ClusterConfigFillers() []api.ClusterFiller {
-	return v.clusterFillers
+	_, ok := os.LookupEnv(ClusterIPPoolEnvVar)
+	var clusterIP string
+	var err error
+	if ok {
+		clusterIP, err = PopIPFromEnv(ClusterIPPoolEnvVar)
```

Comment:
It should be if it is not set or if the string is empty right?

Reply:
Pop ip catches it if its not empty, but I'll add it here as well